.gitignore

```diff
@@ -60,6 +60,7 @@ src/plonetheme.barceloneta/
 src/slc.docconv/
 src/sphinx.themes.plone/
 src/zope.testrunner/
+src/ploneintranet/theme/static/generated/
```

Comment:
This directory stores copies of the HTML files from the prototype referenced by rules.xml, so it shouldn't be ignored!